components/datasciencepipelines/datasciencepipelines.go

```diff
@@ -72,7 +72,7 @@ func (d *DataSciencePipelines) ReconcileComponent(ctx context.Context,
 	enabled := d.GetManagementState() == operatorv1.Managed
 	monitoringEnabled := dscispec.Monitoring.ManagementState == operatorv1.Managed
 
-	platform, err := deploy.GetPlatform(cli)
+	platform, err := deploy.GetPlatform(ctx, cli)
```

Comment:
Do we need this change? I think we should minimize the changes going into backport

Reply:
so the change is in the `isManagedRHODS()` which now require a ctx to do `cli.Get()`
then either in the caller (`GetPlatform()`)  pass down ctx or we have a `context.TODO` in `isManagedRHODS()`

I am open for either way.

Reply:
I did a revert on the context part, now it should be much small updates in the PR @VaishnaviHire

Reply:
Hi thanks ! Yes this looks good to me now